ui/src/app/machines/views/Machines.js

```diff
@@ -3,33 +3,67 @@ import pluralize from "pluralize";
 import React from "react";
 import { useSelector } from "react-redux";
 
-import { machine as machineSelectors } from "app/base/selectors";
+import { useRouter } from "app/base/hooks";
+import {
+  machine as machineSelectors,
+  resourcepool as resourcePoolSelectors
+} from "app/base/selectors";
 import Routes from "app/machines/components/Routes";
 import Section from "app/base/components/Section";
+import Tabs from "app/base/components/Tabs";
 
 const Machines = () => {
   const machines = useSelector(machineSelectors.all);
   const machinesLoaded = useSelector(machineSelectors.loaded);
+  const resourcePools = useSelector(resourcePoolSelectors.all);
+  const { location } = useRouter();
+
+  const machineString = `${machines.length} ${pluralize(
+    "machine",
+    machines.length
+  )}`;
+  const poolString = `${resourcePools.length} ${pluralize(
+    "resource pool",
```

Comment:
In legacy, both "machine" and "resource pool" are capitalised.

Reply:
fixed!